src/current/v23.2/live-migration-service.md

```diff
@@ -0,0 +1,598 @@
+
+The `molt-lms-cli` command-line interface is used to inspect the LMS instances and [perform cutover](#perform-a-cutover).
+
+To install `molt-lms-cli`, [download the binary](https://molt.cockroachdb.com/).
+
+### Commands
+
+|       Command        |                                                                                     Usage                                                                                      |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `initialize`         | Set up the required objects for running the LMS. You must run this before using the LMS.                                                                                       |
+| `connections list`   | List all client connections to the LMS and their most recent queries.                                                                                                          |
+| `cutover consistent` | Specify a [consistent cutover](#consistent-cutover). You must also specify `begin`, `commit`, or `abort`. For usage details, see [Consistent cutover](#consistent-cutover).    |
+| `begin`              | Begin a consistent cutover. This pauses traffic to the source of truth.                                                                                                        |
+| `commit`             | Commit a consistent cutover. This switches the source of truth to the target database.                                                                                         |
+| `abort`              | Abort a consistent cutover after running `consistent cutover begin`, unless you have also run `consistent cutover commit`. This resumes traffic to the source of truth.        |
+| `cutover immediate`  | Initiate an [immediate cutover](#immediate-cutover). This switches the source of truth to the target database. For usage details, see [Immediate cutover](#immediate-cutover). |
+| `status`             | Display the current configuration of the LMS instances.                                                                                                                        |
+
+### Flags
+
+|         Flag         |                                                                                                    Description                                                                                                    |
+|----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--orchestrator-url` | The URL for the orchestrator, using the [configured port](#service-type). This flag is required unless the value is exported as an environment variable using `export CLI_ORCHESTRATOR_URL="{orchestrator-URL}"`. |
```

Comment:
I think it's also worth mentioning that if someone is using SSL certs, their protocol prefix is: `https` versus `http`